gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -307,6 +310,54 @@ case class TransformPreOverrides() extends Rule[SparkPlan] {
     }
   }
 
+  def transformSupportSortWithProjection(plan: SortExec, isSupportAdaptive: Boolean) : SparkPlan = {
```

Comment:
Could you explain why this part of logis is put here instead of in the sort transformer?

Reply:
We need to put the projection before the range partition, we cannot add a range partition action in the `SortExecTransformer`